packages/framework/src/Model/Product/Search/ResultIdsData.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\FrameworkBundle\Model\Product\Search;
+
+class ResultIdsData
+{
+    /**
+     * @var int
+     */
+    protected $total;
+
+    /**
+     * @var array
```

Comment:
```suggestion
     * @var int[]
```

Reply:
applied :)

Reply:
In a e51296776dc6f2def596015daa68a23780b53e23

Reply:
its changed for getter but not for the variable itself